pkgs/development/python-modules/smolagents/default.nix

```diff
@@ -62,6 +62,15 @@ buildPythonPackage rec {
     hash = "sha256-ING+C2MACKFto+1FON5OGFgzLf8SM99ViTdADzNzQLw=";
   };
 
+  # TODO: remove at the next release
+  # ImportError: cannot import name 'require_soundfile' from 'transformers.testing_utils'
+  # Caused by: https://github.com/huggingface/transformers/commit/1ecd52e50a31e7c344c32564e0484d7e9a0f2256
+  # Fixed in: https://github.com/huggingface/smolagents/pull/1625
+  postPatch = ''
+    substituteInPlace tests/test_types.py \
+      --replace-fail "require_soundfile" "require_torchcodec"
```

Comment:
What's `require_torchcodec`? Just some random dummy function to take it's place?

Reply:
I copied what they have done in https://github.com/huggingface/transformers/pull/39156/commits/e207f35e8c8bdfe9f085f76ee3f37a9830d5def2#diff-1063ef75ba73fe97fec48faf71f5020152ca85811784caaef74d4ca18fc6049f.

Reply:
Makes sense.